### ISSUE TITLE
Monterey 12.2.1

### ISSUE THREAD

**Issue by ZJSP** [chris1111/Wireless-USB-Big-Sur-Adapter#179]
Hiya, 
I have Moneterey 12.2.1 and a Archer T2U and for the love of me i cant get the software install be it this version of it or the officeal version which is only supported up to 10.1. 

is there anything i can do or is my virtual machine doomed to a ethernet connection forever. 

all the best 

ZJSP

**Comment by chris1111** [comment]
Gives more infos about what append

**Comment by ZJSP** [comment]
thank you for the quick reply mate. 

I'm running this on a 2019 Macbook Pro on Moneterey 12.2.1, running the V14 of your software.
I have disabled sip security and followed all of the instructions (i believe);
I open the installer and it gets to about 85% throught the installation then it stops on "Running Scripts"  and hits me with a "The Installation failed" "The Installer encounted an error that caused the installation to fail. Contact the software manufacturer" 

I've tried pretty much everything i can think of and it wont have it. 

any help or suggestions massivly appreciated. 

ZJSP
